### PR TITLE
fix: update printing dependencies

### LINKED ISSUE #124: Fix printing on `version-15` branch
`from PyPDF2 import PdfFileWriter` => `from pypdf import PdfWriter`

Also need to refactor to new API for `get_jinja_hooks`

Beam Settings font size should default to 12

### PR DESCRIPTION
**Pull request by Alchez**
Closes #124

### CODE REVIEW

File: beam/beam/doctype/beam_settings/beam_settings.json
Comment (Alchez) [inline beam/beam/doctype/beam_settings/beam_settings.json:1]: @agritheory I'm not sure how the indentation was set to tabs earlier, but it seems like Frappe auto-generated this JSON using spaces. Is there a toggle I'm missing?

Comment (agritheory) [reply beam/beam/doctype/beam_settings/beam_settings.json:1]: Prettier formats JSON. Frappe generates JSON with spaces.

Comment (Alchez) [reply beam/beam/doctype/beam_settings/beam_settings.json:1]: Hmm, I didn't have Prettier change my JSON files (seems like pre-commit only changes JS, Vue and SCSS files).

**Comment by Alchez** [comment]
@agritheory I've added a commit for fixing the test runner

**Review by agritheory — APPROVED** [review]

**Pull request merged**